pkg/api/unversioned/well_known_labels.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned
+
+const LabelZoneFailureDomain = "kubernetes.io/zone-failure-domain"
```

Comment:
@bgrant0607 it was in my head, but I've created an issue now.  Will send a PR today.

https://github.com/kubernetes/kubernetes/issues/17506